lago/utils.py

```diff
@@ -533,12 +534,12 @@ def load_virt_stream(virt_fd):
         dict: Loaded virt config
     """
     try:
-        virt_conf = json.load(virt_fd)
+        virt_conf = json.load(virt_fd, object_pairs_hook=collections.OrderedDict)
     except ValueError:
         virt_fd.seek(0)
         virt_conf = yaml.load(virt_fd)
 
-    return deepcopy(virt_conf)
+    return copy.deepcopy(virt_conf)
```

Comment:
In that if/elif, you should put the OrderedDict elif before the dict one, as an ordereddict is an instance of dict:
```
In [3]: import collections

In [4]: a = col
%colors      collections  

In [4]: a = collections.OrderedDict()

In [5]: isinstance(a, dict)
Out[5]: True
```